pkg/components/machineconfigoperator/component.go

```diff
@@ -39,6 +39,11 @@ var MachineConfigOperatorComponent = Component{
 				IncludeAll:   []string{"Pods cannot access the /config"},
 				Capabilities: []string{"Config"},
 			},
+			{Suite: "Improve MachineConfigPool list table for update status"},
+			{Suite: "MCO"},
```

Comment:
in MCO QE dedicated jobs, only suite `MCO` is used in generated junit xml file e..g [gcp-ipi-f9-longduration-mco-critical/openshift-extended-test-longduration/artifacts/junit/import-MCO.xml](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-gcp-ipi-f9-longduration-mco-critical/1787421969621192704/artifacts/gcp-ipi-f9-longduration-mco-critical/openshift-extended-test-longduration/artifacts/junit/import-MCO.xml), I think other suites can be removed